Updates/841_Saltgurka-Honor_Hold_Updates.sql

```diff
@@ -0,0 +1,1092 @@
+-- ----------------------------------------------------------
+-- Model and equipment updates
+-- ----------------------------------------------------------
+-- Update EquipmentTemplate for all Honor Hold Defenders. Entry 20513 (the two guards next to the destroyed keep) had the correct ones, all others were wrong for some reason.
+UPDATE `creature_template` SET `EquipmentTemplateId`=1195 WHERE `entry`=16842;
+
+-- Also the human model was missing from their creature_template
+UPDATE `creature_template` SET `ModelId1`=16387,`ModelId2`=16388,`ModelId3`=16389,`ModelId4`=16390 WHERE `entry` IN (16842,20513); -- 16390

```

Comment:
I assume the reason the guards next to the keep (previously entry 20513) only had the human model is because their speech to each other specifically mentions them being "grumpy old men". 

Why the other guards only had the dwarf model, I don't know. But they should definitely be random between those 4 models. You can see all of them at 0:12 in this video: https://youtu.be/3jUQIS234fQ?t=12s